fix --since to wait_for_docker_container_task example

YannCharlou-CleverAge opened the pull request:
to avoid checking the logs from a previous container start

Fix my previous PR to use a real RFC3339 date for the --since docker command option

lyrixx commented on examples/wait_for.php:
```diff
@@ -99,13 +99,14 @@ function custom_wait_for_task(string $thing = 'foobar'): void
 function wait_for_docker_container_task(): void
 {
     try {
+        $checkLogSince = (new \DateTimeImmutable())->format(\DateTimeInterface::RFC3339);
```
You can use the date fonction here

YannCharlou-CleverAge replied:
You are right. Just fixed in the last commit. :-)

pyrech reviewed (approved):
Thanks

YannCharlou-CleverAge commented:
> @YannCharlou-CleverAge Could you rebase your PR on top of branch main and fix the CS so we can merge it?

Sorry for the delay I was very busy these days.

- coding standards : fixed
- branch : rebased

lyrixx reviewed (approved):
Thanks
